internal/tests/SchemaVersionResource/schema_version_resource_avro_test.go

```diff
@@ -22,25 +23,69 @@ func TestSchemaVersionAvroResource(t *testing.T) {
+			{
+				Config: GetProvider() + GetFile("axual_schema_version_avro_multiple_versions_for_same_schema.tf"),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "description", "Gitops test schema version2"),
-					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "version", "2.0.0"),
-					CheckBodyMatchesFile("axual_schema_version.axual_gitops_test_schema_version1", "body", "avro-schemas/gitops_test_v2_backwards_compatible.avsc"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "version", "1.0.0"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "description", "Gitops test schema version"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "full_name", "io.axual.qa.general.GitOpsTest1"),
+					CheckBodyMatchesFile("axual_schema_version.axual_gitops_test_schema_version1", "body", "avro-schemas/gitops_test_v1.avsc"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version2", "version", "2.0.0"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version1", "description", "Gitops test schema version"),
+					resource.TestCheckResourceAttr("axual_schema_version.axual_gitops_test_schema_version2", "full_name", "io.axual.qa.general.GitOpsTest1"),
+					CheckBodyMatchesFile("axual_schema_version.axual_gitops_test_schema_version2", "body", "avro-schemas/gitops_test_v2_backwards_compatible.avsc"),
+					resource.TestCheckResourceAttrPair(
+						"axual_schema_version.axual_gitops_test_schema_version1", "description",
+						"axual_schema_version.axual_gitops_test_schema_version2", "description",
+					),
+					resource.TestCheckResourceAttrPair(
+						"axual_schema_version.axual_gitops_test_schema_version1", "full_name",
+						"axual_schema_version.axual_gitops_test_schema_version2", "full_name",
+					),
+					// to verify that the both schema versions belong to the same schema
+					resource.TestCheckResourceAttrPair(
+						"axual_schema_version.axual_gitops_test_schema_version1", "schema_id",
+						"axual_schema_version.axual_gitops_test_schema_version2", "schema_id",
+					),
 				),
 			},
 			{
-				Config: GetProvider() + GetFile("axual_schema_version_avro_v3_replaced.tf"),
+				// To ensure cleanup if one of the test cases had an error
+				Destroy: true,
+				Config:  GetProvider() + GetFile("axual_schema_version_avro_v3_replaced.tf"),
+			},
+		},
+	})
+}
+
+func TestSchemaVersionAvroWithOwnersResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: GetProviderConfig(t).ProtoV6ProviderFactories,
+		ExternalProviders:        GetProviderConfig(t).ExternalProviders,
+
+		Steps: []resource.TestStep{
+			{
+				Config: GetProvider() + GetFile("axual_schema_version_with_owner.tf"),
```

Comment:
Need to also update this line:

`{
				// To ensure cleanup if one of the test cases had an error
				Destroy: true,
				Config:  GetProvider() + GetFile("axual_schema_version_with_owner.tf"),
			},`